routes/documents.js

```diff
@@ -2,56 +2,61 @@ const express = require('express');
 
 const router = express.Router();
 const Document = require('../models/document');
+const User = require('../models/user');
 const { DocumentBodySchema } = require('./validators/validators');
 const validator = require('express-joi-validator');
 
-router.get('/', (req, res, next) => {
-  Document.find({})
-    .select('-content')
-    .then((document) => {
-      res.send(document);
+const sendError = res =>
+  res.status(401).send({
+    message: 'User is not logged!',
+    status: 401,
+  });
+
+
+const isAuthenticated = (req, res, next) =>
+  User.findById(req.signedCookies['simul-doc'])
+    .then((user) => {
+      if (!user) return sendError(res);
```

Comment:
Do not explicitly send error. Just create an error here and pass it to `next()` function.